base/utferror.jl

```diff
@@ -0,0 +1,30 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+##\brief      Error messages for Unicode / UTF support
```

Comment:
Sure, I think it's completely reasonable to settle on some conventions eventually. I think @catawbasam is right about side issues, though, so let's keep that discussion focussed in #8966 so that things can move forward here.